### PR TITLE
Fix mono package versions during stabilization

### PR DESCRIPTION
**Pull request by ViktorHofer**
The issues that are being fixed via this PR happened and were fixed during the 7.0 stabilization release exercise.

This will also need to be backported into release/7.0-rc1.

### CODE REVIEW

**Comment by ViktorHofer** [comment]
cc @akoeplinger

**Review by carlossanlop — APPROVED** [review]

**Review by steveisok — APPROVED** [review]

File: src/mono/nuget/Directory.Build.targets [hunk 1]
```diff
@@ -1,5 +1,13 @@
 <Project>
   <UsingTask TaskName="GenerateFileFromTemplate" AssemblyFile="$(WorkloadTasksAssemblyPath)" />
   <Import Project="..\Directory.Build.targets" />
+  
+  <PropertyGroup>
+    <!-- Central place to set the versions of all mono pkgprojs. -->
+    <PackageVersion Condition="'$(PackageVersion)' == ''">$(ProductVersion)</PackageVersion>
+    <StableVersion Condition="'$(StabilizePackageVersion)' == 'true' and '$(StableVersion)' == ''">$(PackageVersion)</StableVersion>
```
Comment (akoeplinger) [inline src/mono/nuget/Directory.Build.targets:8]: the coreclr version of this also checks for `'$(PreReleaseVersionLabel)' != 'servicing'` do we need this here too?

https://github.com/dotnet/runtime/blob/cb52e3b88177cb86929b829d92b03561ad4e806b/src/coreclr/.nuget/Directory.Build.targets#L7

**Review by akoeplinger — APPROVED** [review]

File: src/mono/nuget/Directory.Build.targets (hunk 1 above)
Comment (ViktorHofer) [reply src/mono/nuget/Directory.Build.targets:8]: No, as the coreclr packages intentionally don't ship during servicing which isn't true for the mono packages.

**Pull request merged**